src/navigation/_CollapsingNavigationMenu.jsx

```diff
@@ -150,42 +279,45 @@ const CollapsingNavigationMenu = ({ selectedPath = undefined, menuItems, onSelec
         selectedRef = selectedItem;
       }
 
+      visibleNodes.push({ id, parent });
       const menuItemClassNames = classNames(
         cx([
           'item',
           { 'is-selected': isSelected },
         ]),
       );
 
+      const optionalAttributes = itemHasChildren ? { 'aria-expanded': itemIsOpen, 'aria-haspopup': true } : {};
+
       return (
         <React.Fragment key={item.path}>
           <div className={!firstLevel ? cx('indent') : null}>
             <div
               className={menuItemClassNames}
               tabIndex="0"
```

Comment:
Only the currently selected navigation tree item should have a tabindex of '0', the others should have '-1' set with focus managed only with arrow keys.

Reply:
just made a change to address this issue.